### DISCUSSION TITLE
Keybinding to select tab by number?

### DISCUSSION THREAD

**Discussion by maximilianschmitt** [zed-industries/zed#6499] [Feature Requests]
In Chrome and most other tabbed apps, I can use CMD+1, CMD+2 to select the first, second tabs, etc.

I notice that in Zed, CMD+2/3 splits the editor instead. How can I change this behaviour? I couldn't find the binding in the keymaps.json file.

Thanks!

**Comment by Yevgnen** [comment]
Same question. And those keys can not be rebound in neither "Pane" and "Workspace" context.

**Comment by maxhoffmann** [comment]
Adding this to `keymap.json` solved it for me:

```json
[
  {
    "context": "Pane",
    "bindings": {
      "cmd-1": ["pane::ActivateItem", 0],
      "cmd-2": ["pane::ActivateItem", 1],
      "cmd-3": ["pane::ActivateItem", 2],
      "cmd-4": ["pane::ActivateItem", 3],
      "cmd-5": ["pane::ActivateItem", 4],
      "cmd-6": ["pane::ActivateItem", 5],
      "cmd-7": ["pane::ActivateItem", 6],
      "cmd-8": ["pane::ActivateItem", 7],
      "cmd-9": ["pane::ActivateItem", 8]
    }
  }
]
```

**Comment by SomeoneToIgnore** [comment]
Zed actually does provide `ctrl-number` default bindings for those, in case it suits you too.
